src/layouts/sidebar-sidecar/components/table-of-contents/use-active-section.ts

```diff
@@ -18,6 +20,20 @@ const getFullNavHeaderHeight = () => {
     { asNumber: true }
   ) as number
 
+  /**
+   * We appear to be seeing intermittent errors from '--navigation-header-height' being resolved
+   * to an empty string on navigation. Adding a warning and a fallback
+   */
+  if (Number.isNaN(navigationHeaderHeight)) {
+    if (process.env.NODE_ENV === 'development') {
+      console.warn(
+        'Unable to read --navigation-header-height, falling back to static value.'
+      )
+    }
+
+    return NAVIGATION_HEADER_HEIGHT_FALLBACK
+  }
```

Comment:
💭 Thoughts on putting this in `getCSSVariableFromDocument` instead and adding a `fallbackValue` option?

Reply:
Sure! Updated in https://github.com/hashicorp/dev-portal/pull/601/commits/3f041cba52c18731f34b727b410cfac7acd5e703